terraform/modules/stack/base/base.tf

```diff
@@ -15,6 +15,7 @@ module "vpc" {
   nat_gateway_instance_type         = var.nat_gateway_instance_type
   monitoring_security_groups        = var.target_monitoring_security_groups
   concourse_security_groups         = var.target_concourse_security_groups
+  credhub_security_groups           = var.target_credhub_security_groups
```

Comment:
the terraform check is complaining about this. I think you need to update the vpc module to know about this variable?

Reply:
It looks like I missed this: https://github.com/cloud-gov/cg-provision/pull/807/commits/e9eb960a25e05ee18aec397de90dfa3f2f7b864e#diff-6b3bfbeef0b5f97151e8df46b14b1ac7913ad73d151fe8afe19c60c5989f7440R114

There is also this concourse reference but it is in k8s module. I can't imagine this actually needs to be there so I am not adding credhub to it. https://github.com/cloud-gov/cg-provision/blob/39c50d95dfb77ec0e13f00d28f1fb944badaa21a/terraform/stacks/main/stack.tf#L173